lib/install.js

```diff
@@ -947,6 +947,17 @@ function clean() {
     return config.save();
   });
 }
+function clean() {
+  return new Promise(function(resolve, reject) {
+    fs.lstat(config.pjson.packages, function(err, stats) {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(stats.isSymbolicLink() ? true : _clean());
+      }
+    });
+  });
```

Comment:
We don't need to skip the entire clean, we just need to skip steps that alter `jspm_packages`, which is 4 and 5 above.